app/src/main/java/fr/free/nrw/commons/navtab/MoreBottomSheetFragment.java

```diff
@@ -194,32 +191,27 @@ private void sendFeedback() {
         }
     }
 
-    @OnClick(R.id.more_about)
-    public void onAboutClicked() {
+    protected void onAboutClicked() {
         final Intent intent = new Intent(getActivity(), AboutActivity.class);
         intent.addFlags(Intent.FLAG_ACTIVITY_REORDER_TO_FRONT | Intent.FLAG_ACTIVITY_SINGLE_TOP);
-        getActivity().startActivity(intent);
+        requireActivity().startActivity(intent);
```

Comment:
Several different instances - the call to `requireActivity()` replaces `getActivity()` to add an implicit (Android supplied) null check.